### PR TITLE
prepare tough-kms and tuftool releases

### PR DESCRIPTION
**Pull request by webern**
*Issue #, if available:*

#262

*Description of changes:*

- Update deps in the project.
- Bump version at the patch level and update the changelog to release `tough-kms` v0.1.1.
- Bump version at the 'major-ish' level and update the changelog to release `tuftool` v0.6.0.

*Testing*

 - [x] Ran my loop that signs and then loads repos using KMS, it's still working.
 - [x] Ran tuftool tests with `--features integ`.
 - [x] All the CI tests.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by webern** [comment]
I'm going to add a commit to this to prep tuftool release as well.

File: tuftool/CHANGELOG.md [hunk 1]
```diff
@@ -78,6 +93,7 @@ Major update: much of the logic in `tuftool` has been factored out and added to
 ### Added
 - Everything!
 
+[0.6.0]: https://github.com/awslabs/tough/compare/tuftool-v0.5.0...tuftool-v0.6.0
```
Comment (zmrow) [inline tuftool/CHANGELOG.md:96]: We'll also need to bump the version of tuftool in `tuftool/Cargo.toml`

Comment (webern) [reply tuftool/CHANGELOG.md:96]: Good catch.

**Review by srgothi92 — APPROVED** [review]
LGTM

**Review by zmrow — APPROVED** [review]
:vulcan_salute:

File: tuftool/Cargo.toml [hunk 2]
```diff
@@ -35,13 +35,13 @@ simplelog = "0.8.0"
 snafu = { version = "0.6.8", features = ["backtraces-impl-backtrace-crate"] }
 structopt = "0.3"
 tempfile = "3.1.0"
-tokio = "0.2.21"
+tokio = "0.3.3"
```
Comment (iliana) [inline tuftool/Cargo.toml]: Are we sure this works, given that Rusoto is still on tokio 0.2? (I don't know the compatibility story here.)

Comment (webern) [reply tuftool/Cargo.toml]: It does based on testing. We can back it out if concerned though.

There is a change to the interface for tokio::Runtime::new, and the tokio 0.3 breaks up the tokio project into features in a way that it didn't before. But in our build/testing here it works.

**Comment by webern** [comment]
> webern force-pushed the webern:tough-kms-release-0.1.1 branch from 5f9f4fb to a520b5e

Back-out the tokio upgrade to be on the safe-side.

File: tuftool/Cargo.toml (hunk 2 above)
Comment (webern) [reply tuftool/Cargo.toml]: Upon further reflection, you're right, too risky. Backed that out, CI running...

**Review by iliana — APPROVED** [review]

**Pull request merged**